components/brave_rewards/browser/test/common/rewards_browsertest_response.cc

```diff
@@ -324,11 +321,7 @@ void RewardsBrowserTestResponse::Get(
     return;
   }
 
-  if (URLMatches(
-      url,
-      "/order",
-      PREFIX_V1,
-      ServerTypes::kPayments)) {
+  if (url.find("/v1/orders") != std::string::npos) {
```

Comment:
The change from `order` to `orders` is intentional here?

Reply:
yes, api is `orders`, it was wrong before